cmd/helpMessages.go

```diff
@@ -305,8 +305,15 @@ Set the environment variable AZCOPY_SPA_CERT_PASSWORD to the certificate's passw
 
    Please treat /path/to/my/cert as a path to a PEM or PKCS12 file-- AzCopy does not reach into the system cert store to obtain your certificate.
    --certificate-path is mandatory when doing cert-based service principal auth.
+
+Subcommand for login to check the login status of your current session.
+	- azcopy login status 
```

Comment:
What are your thoughts on having this as additional flag? 'azcopy login --status', this would be much smaller change in that case. Adding @manumsft here

Reply:
I agree with this. It'll reduce the amount of code changes to bare minimum.

Reply:
I agree @nakulkar-msft . It makes sense. thanks.